### PR TITLE
Repository in memory

### PR DESCRIPTION
**Pull request by JaimeSalas**

### CODE REVIEW

File: src/repositories/channelRepository.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+};
+
+exports.existUser = async (channelKey, userId) => {
+    const existChannel = await exports.existChannel(channelKey);
+    return new Promise((resolve, reject) => {
+        if (!existChannel) {
+            reject(`${channelKey} does not exist.`);
+        };
+        resolve(
+            !!channels[channelKey].users.find(u => u === userId)
```
Comment (brauliodiez) [inline src/repositories/channelRepository.js]: Are we covering here the cae where channelKey is an empty string?

File: test/channelRepository.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            const message = {
+                channelKey: 'comics',
+                userId: 'jai',
+                text: 'batman is awesome',
+            };
+            await channelRepository.addMessage(message);
+            const messages = await channelRepository.getMessages(message.channelKey);
+            expect(messages.length).to.greaterThan(0);
+        });
+    });
```
Comment (brauliodiez) [inline test/channelRepository.test.js]: Would it be a good idea to create tests for empty string, and maybe userId or ChannelKey null or undefined?

**Comment by brauliodiez** [comment]
Good stuff just spotted one minor thing

File: src/repositories/channelRepository.js (hunk 1 above)
Comment (JaimeSalas) [reply src/repositories/channelRepository.js]: I've added new tests for this.

**Comment by brauliodiez** [comment]
Good stuff, approved from my side

**Pull request merged**